filebeat/input/tcp/config.go

```diff
@@ -0,0 +1,25 @@
+package tcp
+
+import (
+	"time"
+
+	"github.com/elastic/beats/filebeat/harvester"
+)
+
+type config struct {
+	harvester.ForwarderConfig `config:",inline"`
+	Host                      string        `config:"host"`
+	LineDelimiter             string        `config:"line_delimiter" validate:"nonzero"`
+	Timeout                   time.Duration `config:"timeout" validate:"nonzero,positive"`
+	MaxMessageSize            int64         `config:"max_message_size" validate:"nonzero,positive"`
+}
+
+var defaultConfig = config{
+	ForwarderConfig: harvester.ForwarderConfig{
+		Type: "tcp",
+	},
+	LineDelimiter:  "\n",
+	Host:           "localhost:9000",
+	Timeout:        time.Second * 5 * 60,
```

Comment:
Why don't you write `5 * time.Minute` instead of calculating it?

Reply:
I can and will.